tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/ReleasePlanTool/ReleasePlanTool.cs

```diff
@@ -186,5 +214,118 @@ Please create a pull request in the public Azure/azure-rest-api-specs repository
                 return $"Failed to create release plan work item: {ex.Message}";
             }
         }
+
+        [McpServerTool, Description("Update the SDK Info in the release plan work item. This tool is called to update SDK language, package name, optional language exclusion note in the release plan work item." +
+            " sdkDetails parameter is a JSON of list of SDKInfo and each SDKInfo contains Language and PackageName as properties.")]
+        public async Task<string> UpdateReleasePlanSDKInfo(int releasePlanWorkItemId, string sdkDetails, string languageExclusionNote = "")
+        {
+            try
+            {
+                if (releasePlanWorkItemId <= 0)
+                {
+                    return "Invalid release plan ID.";
+                }
+
+                if (string.IsNullOrEmpty(sdkDetails))
+                {
+                    return "No SDK information provided to update the release plan.";
+                }
+
+                // Fix for CS8600: Ensure sdkDetails is not null before deserialization
+                List<SDKInfo>? SdkInfos = JsonSerializer.Deserialize<List<SDKInfo>>(sdkDetails);
+                if (SdkInfos == null)
+                {
+                    return "Failed to deserialize SDK details.";
+                }
+
+                // Validate SDK language name
+                if (SdkInfos.Any(sdk => !supportedLanguages.Contains(sdk.Language, StringComparer.OrdinalIgnoreCase)))
+                {
+                    return $"Unsupported SDK language found. Supported languages are: {string.Join(", ", supportedLanguages)}";
+                }
+
+                var updated = await devOpsService.UpdateReleasePlanSDKDetails(releasePlanWorkItemId, SdkInfos, languageExclusionNote);
+                if (!updated)
+                {
+                    SetFailure();
+                    return "Failed to update release plan with SDK details.";
+                }
+                else
+                {
+                    StringBuilder sb = new("Updated SDK details in release plan.");
+                    sb.AppendLine();
+                    foreach (var sdk in SdkInfos)
+                    {
+                        sb.AppendLine($"Language: {sdk.Language}, Package name: {sdk.PackageName}");
+                    }
+                    return output.Format(sb.ToString());
+                }
+            }
+            catch (Exception ex)
+            {
+                SetFailure();
+                return $"Failed to update release plan with SDK details: {ex.Message}";
+            }
+        }
+
+        [McpServerTool, Description("Link package namespace approval issue to release plan. This requires GitHub issue URL for the namespace approval request and release plan work item id.")]
```

Comment:
this is true only for management plane. Will adding that to the description will be enough so data plane folks don't get confused?

Reply:
Correct. This is handled in the prompt to trigger it only for management plane spec.  I will update the description to note it.

Reply:
fixed in https://github.com/Azure/azure-sdk-tools/pull/10834